l10n_it_fatturapa_pec/views/fetchmail_view.xml

```diff
@@ -15,6 +15,9 @@
                 <!-- Hiding, because it would not be considered. See 'fetch_mail' override -->
                 <attribute name="attrs">{'invisible': [('is_fatturapa_pec', '=', True)]}</attribute>
             </xpath>
+            <field name="active" position="after">
+                <field name="e_inv_notify_partner_ids" widget="many2many_tags"/>
```

Comment:
The field is used only if the server is `is_fatturapa_pec`, maybe it makes sense to show/hide it properly with `{'invisible': [('is_fatturapa_pec', '=', False)]}`

Maybe this can be the beginning of a new tab/section dedicated to e-invoice

Reply:
@SimoRubi thanks, done